shell/probe/raindrop.ps1

```diff
@@ -1,10 +1,10 @@
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
-function Execute { 
-    Param([String]$File)
+function Execute {
+    Param([String]$File, [String]$StdoutFile, [String]$StderrFile)
 
     try {
-        $R = (Start-Process -FilePath $File -Wait -NoNewWindow -PassThru).ExitCode
+        $R = (Start-Process -FilePath $File -Wait -NoNewWindow -PassThru -RedirectStandardOutput $StdoutFile -RedirectStandardError $StderrFile).ExitCode
```

Comment:
Since we're opening this up, maybe settle on variable naming conventions? the rest seem to be lowercase words, not uppercase single letters.

Reply:
function params are uppercase and everything else is lowercase. No idea why but that's what was in here and I googled it and that does seem to be the powershell convention.